src/main/java/com/flightstats/hub/dao/aws/HubS3Client.java

```diff
@@ -66,7 +68,7 @@ InitiateMultipartUploadResult initiateMultipartUpload(InitiateMultipartUploadReq
         try {
             return s3Client.initiateMultipartUpload(request);
         } catch (SdkClientException e) {
-            countError(e, request, "initiateMultipartUpload", Arrays.asList("bucket:" + request.getBucketName(), "key:" + request.getKey()));
+            countError(e, request, "initiateMultipartUpload", Collections.singletonList(request.getKey()));
```

Comment:
FWIW, with JDK 9+ you can use `List.of(...)` and it's smart about optimizing for the number of passed elements.

Reply:
I keep forgetting about that and letting IntelliJ convert from `asList`.

Reply:
And also forgetting that that hub uses Java 8. :(